tsconfig.eslint.json

```
@@ -1,5 +1,13 @@
 {
   "extends": "./tsconfig.json",
+  "include": [
```

Biome (2.5.5)

[error] The key include was already declared.

(lint/suspicious/noDuplicateObjectKeys)